uiowa/uiowa
# Uninstall `oembed_providers` module and associated code/config changes

Uninstalls the `oembed_providers` module and removes the associated code/config. This change means that we are reverting back to the core functionality for this.

Resolves #4533 

# How to test

```
ddev blt ds --site sandbox.uiowa.edu
```
- https://sandbox.uiowa.ddev.site/media-kaltura/kalturamediaspace-small-captions - See that nothing renders except the caption where the kKaltura media entities have been placed
```
ddev drush @sandbox.local uli
```
- Test editing existing media embeds (changing style) for YouTube and Vimeo in a WYSIWYG field (like body)
- Test deleting existing media embeds for YouTube and Vimeo in a WYSIWYG field (like body)
- Test adding new media embeds for YouTube and Vimeo in the body field
- https://sandbox.uiowa.ddev.site/media/remotevideo/121 - Delete the Kaltura video
- https://sandbox.uiowa.ddev.site/media-kaltura/kalturamediaspace-small-captions - Confirm that the missing media entity widget is rendering for the embeds referencing the media entity you just deleted
